.teamcity/subprojects.json

```diff
@@ -681,6 +681,13 @@
   {
     "name": "plugins-java",
     "path": "platforms/jvm/plugins-java",
+    "unitTests": false,
```

Comment:
Probably shouldn't address it in this PR, but it seems there are a number of tests which should be moved into this project from `:plugins`

Reply:
Just did a pass, hopefully got them all sorted this time.